sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -102,7 +102,7 @@
           "value": ""
         }
       ],
-      "content": "Please ensure your information below matches your identity verification form.",
+      "content": "Please ensure your profile information exactly matches the information you provided on the identity verification form issued to you by StudentAid BC.",
```

Comment:
This form is common between student profile setup(as BCSC user) and BCeid user. 

The story is particular to basic bceid create profile. Hence changing here would change the content in both the places. 

So in the ACs we need to be sure of what is common to both pages and what is very specific to Basic bceid.

![image](https://github.com/user-attachments/assets/7f48e18d-9506-4964-81d8-76b453573440)

![image](https://github.com/user-attachments/assets/76be312b-890a-4e92-8564-76bcedb442bb)